packages/fungible-token/bridge-fungible-token/interface/src/bridge.sw

```diff
@@ -35,4 +36,6 @@ abi Bridge {
 
     #[storage(read)]
     fn asset_to_l1_address(asset_id: AssetId) -> b256;
+
+    fn double_value(foo: u64) -> u64;
```

Comment:
Why is this needed?

Reply:
A debugging function back from when LDC was still semi functional. Removed